community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/CostBasedExecutablePlanBuilder.scala

```diff
@@ -89,8 +89,8 @@ case class CostBasedExecutablePlanBuilder(monitors: Monitors,
 
     val pipeBuildContext = PipeExecutionBuilderContext(metrics.cardinality, semanticTable, plannerName)
 
-    //Check for notifications
-    checkForUnresolvedTokens(ast, semanticTable).foreach(notificationLogger += _)
+    //Check for unresolved tokens for read-only queries
```

Comment:
I though it was annoying to warn the user when doing `CREATE (m: Person)` when `Person` is not in the database. The user is about to create it and it felt noisy to throw a warning in this case. The other reason is that there are tests asserting that all the guides in the browser runs without giving notifications, and these guides typically starts from an empty database and create nodes.  I am not sure what is the correct thing here.